Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -142,6 +148,10 @@ private class ModelCapturingOutputPrinter : OutputPrinter {
 
       public void Inform(string s, TextWriter tw) {
         logger.LogInformation(s);
+        var match = Regex.Match(s, "^Verifying .+[.](?<name>[^.]+) ...$");
+        if (match.Success) {
+          progressReporter.ReportProgress($"Verifying {match.Groups["name"].Value}...");
```

Comment:
I don't like it that the message now contains the verification status and overrides it(`Verifying`). To me the compilation status should indicate what is happening, and the "message" should be additional information/arguments, not something that overrides the status itself, e.g. for maintainability or even down the road consistent translations.
After a second look, it looks like @camrein  already had the same remark here:
https://github.com/dafny-lang/ide-vscode/pull/104/files#r772466907
I second it.

Reply:
I thought we'd do that with more structured messages: for now we were starting with the prover telling the IDE exactly what to display, and then we'd move to more structured messages.  (If we were to consider translations, I would expect the translation to happen once in Dafny, not separately in each IDE).  But I'm happy o change if you want.

Reply:
It's not just for translation (there is no need for that) It's more for separation of concerns. The message "Verifying" should be there only once for semantic reasons.
Please do this change so that the message is just an argument.

Reply:
Done